### ISSUE TITLE
Access is Denied

### ISSUE THREAD

**Issue by d0m1n1c** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#511]
When I run the run.py file, it opens the cmd for a second and flashes an error, then closes.
The Error:

```
FileNotFoundError: [WinError 3] The system cannot find the path specified: 'C:\\Windows\\system32\\analytics\\MyTwitchUsername'

PermissionError: [WinError 5] Access is denied: 'C:\\Windows\\system32\\analytics'
```

Not sure what to do

**Comment by heyimrendy** [comment]
Put the program on different folder, not admin required folder

**Comment by heyimrendy** [comment]
Tested with my windows machine, its normal for me analytics folder created inside repo folder

**Comment by ColinShark** [comment]
Firstly, try to open the `run.py` with an interactive shell, not by double-clicking:  
In the directory of the project (for example `C:/Users/<username>/Twitch-Channel-Points-Miner-v2/`), click "File" in the very top left for a dropdown, then hit "Open CMD/Powershell". Then enter `python run.py`. This will start the program from the current directory. Maybe this helps.

Alternatively, also set up a Virtual Environment with `python -m venv venv`, then activate that with `./venv/Scripts/activate`. Now all packages you install will be inside this environment, not interfering with system wide packages.

**Comment by d0m1n1c** [comment]
> Firstly, try to open the `run.py` with an interactive shell, not by double-clicking: In the directory of the project (for example `C:/Users/<username>/Twitch-Channel-Points-Miner-v2/`), click "File" in the very top left for a dropdown, then hit "Open CMD/Powershell". Then enter `python run.py`. This will start the program from the current directory. Maybe this helps.


Yes this worked! 

tl;dr: File > Windows Powershell (administrator), then  `python run.py`
